pipyadc/pipyadc.py

```diff
@@ -187,7 +187,7 @@ def stop(self):
     def stop_close_all(self):
         """Close all open pigpio SPI handles and stop pigpio connection
         """
-        for handle in reversed(self.open_spi_handles):
+        for handle in reversed(list(self.open_spi_handles.values())):
```

Comment:
I would assume that even before Python 3.6, the reversed() built-in-function would work without turning the dict.values() (or, rather dict.keys()) iterator into a list first.

Can you please check if the following works:
`for handle in reversed(self.open_spi_handles.keys()):`